Author: John Lightsey <lightsey@debian.org>
Description: Fix several minor spelling errors in module and matching documentation.
Bug: https://rt.cpan.org/Ticket/Display.html?id=59091
Forwarded: yes
--- a/README
+++ b/README
@@ -78,13 +78,13 @@
     average manager or secretary should be able to look at and make sense
     of.
 
-    YAML::Tiny does not generate comments, it won't necesarily preserve the
-    order of your hashes, and it will normalise if reading in and writing
+    YAML::Tiny does not generate comments, it won't necessarily preserve the
+    order of your hashes, and it will normalize if reading in and writing
     out again.
 
     It only supports a very basic subset of the full YAML specification.
 
-    Usage is targetted at files like Perl's META.yml, for which a small and
+    Usage is targeted at files like Perl's META.yml, for which a small and
     easily-embeddable module is extremely attractive.
 
     Features will only be added if they are human readable, and can be
@@ -117,8 +117,8 @@
     structure dumps.
 
     Many specification elements that add flexibility or extensibility are
-    intentionally removed, as is support for complex datastructures, class
-    and object-orientation.
+    intentionally removed, as is support for complex data structures,
+    classes and object-orientation.
 
     In general, the YAML Tiny language targets only those data structures
     available in JSON, with the additional limitation that only simple keys
@@ -126,7 +126,7 @@
 
     As a result, all possible YAML Tiny documents should be able to be
     transformed into an equivalent JSON document, although the reverse is
-    not necesarily true (but will be true in simple cases).
+    not necessarily true (but will be true in simple cases).
 
     As a result of these simplifications the YAML Tiny specification should
     be implementable in a (relatively) small amount of code in any language
@@ -198,7 +198,7 @@
     YAML Tiny parsers and emitters are not expected to recover from adapt to
     errors. The specific error modality of any implementation is not
     dictated (return codes, exceptions, etc) but is expected to be
-    consistant.
+    consistent.
 
   4. Syntax
     Character Set
@@ -298,7 +298,7 @@
     Comments work as per the YAML specification in all simple cases. Support
     for indented multi-line comments is not required.
 
-    Seperation spaces work as per the YAML specification in all cases.
+    Separation spaces work as per the YAML specification in all cases.
 
     YAML Tiny Character Stream
 
@@ -331,7 +331,7 @@
 
     Support for the document boundary market "..." is not required.
 
-    If necesary, a document boundary should simply by indicated with a "---"
+    If necessary, a document boundary should simply by indicated with a "---"
     marker, with not preceding "..." marker.
 
     Support for empty streams (containing no documents) is required.
@@ -449,7 +449,7 @@
     Turn Perl data into YAML. This function works very much like
     Data::Dumper::Dumper().
 
-    It takes a list of Perl data strucures and dumps them into a serialized
+    It takes a list of Perl data structures and dumps them into a serialized
     form.
 
     It returns a string containing the YAML stream.
--- a/lib/YAML/Tiny.pm
+++ b/lib/YAML/Tiny.pm
@@ -724,13 +724,13 @@
 your average manager or secretary should be able to look at and make
 sense of.
 
-L<YAML::Tiny> does not generate comments, it won't necesarily preserve the
-order of your hashes, and it will normalise if reading in and writing out
+L<YAML::Tiny> does not generate comments, it won't necessarily preserve the
+order of your hashes, and it will normalize if reading in and writing out
 again.
 
 It only supports a very basic subset of the full YAML specification.
 
-Usage is targetted at files like Perl's META.yml, for which a small and
+Usage is targeted at files like Perl's META.yml, for which a small and
 easily-embeddable module is extremely attractive.
 
 Features will only be added if they are human readable, and can be written
@@ -843,7 +843,7 @@
 
 YAML Tiny parsers and emitters are not expected to recover from adapt to
 errors. The specific error modality of any implementation is not dictated
-(return codes, exceptions, etc) but is expected to be consistant.
+(return codes, exceptions, etc) but is expected to be consistent.
 
 =head2 4. Syntax
 
@@ -944,7 +944,7 @@
 Comments work as per the YAML specification in all simple cases.
 Support for indented multi-line comments is B<not> required.
 
-Seperation spaces work as per the YAML specification in all cases.
+Separation spaces work as per the YAML specification in all cases.
 
 B<YAML Tiny Character Stream>
 
@@ -977,7 +977,7 @@
 
 Support for the document boundary market "..." is B<not> required.
 
-If necesary, a document boundary should simply by indicated with a
+If necessary, a document boundary should simply by indicated with a
 "---" marker, with not preceding "..." marker.
 
 Support for empty streams (containing no documents) is required.
